src/components/users/partials/modal/GroupDetails.tsx

```diff
@@ -98,6 +98,7 @@ const GroupDetails: React.FC<{
 						{/* Navigation buttons and validation */}
 						<WizardNavigationButtons
 							formik={formik}
+							previousPage={() => close()}
```

Comment:
Can be simplified to (in all three instances):

```suggestion
							previousPage={close}
```

Reply:
done, i had copied and pasted from elsewhere in the code :wink: